src/Docker.jl

```diff
@@ -1,4 +1,12 @@
-struct DockerExecutor <: SandboxExecutor
+using Random
+
+mutable struct DockerExecutor <: SandboxExecutor
```

Comment:
Out of curiosity, why is this mutable?

Reply:
Ah, I used to have a finalizer on this, before I decided to make it explicitly cleaned up via `cleanup()`.  Thanks for catching that.